File: 13. Rekursi Lanjut/B. Menggambar Pegunungan/b_menggambar_pegunungan.java
import java.util.Scanner;

public class Solution {
    public static void draw(int N) {
        if (N == 1) {
            System.out.println("*");
        } else {
            draw(N - 1);
            for (int i = 0; i < N; i++) {
                System.out.print("*");
            }
            System.out.println();
            draw(N - 1);
        }
    }

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int N = scan.nextInt();
        scan.close();
        draw(N);
    }
}

File: 13. Rekursi Lanjut/B. Menggambar Pegunungan/b_menggambar_pegunungan.go
package main

import (
	"fmt"
)

func draw(N int) {
	if N == 1 {
		fmt.Println("*")
	} else {
		draw(N - 1)
		for i := 0; i < N; i++ {
			fmt.Print("*")
		}
		fmt.Println()
		draw(N - 1)
	}
}
func main() {
	var N int
	fmt.Scan(&N)
	draw(N)
}
